### ISSUE TITLE
Curios causes JEI to not show up

### ISSUE THREAD

**Issue by Ceimnithe** [TheIllusiveC4/Curios#158]
Heya,
So I just went through the process of building myself a modded 1.16.5, and as I was beginning a playthrough I noticed that somewhere along the way, my JEI interface vanished. Through process of elimination I narrowed it down to Curios, unfortunately. I haven't seen anybody else with this issue, so I could just be an idiot lol. I rolled back to a previous version of JEI to see if the latest JEI update was the instigator, but no luck - as long as Curios is installed, the JEI interface just goes poof. If this is a JEI issue however, let me know and I'll pack up and head over there to whine instead lol.

Thanks in advance for any help!

Versions:
Minecraft: 1.16.5
Curios: 4.0.5.2
JEI: 7.7.1.118
Forge: 36.2.0

![with curios](https://user-images.githubusercontent.com/88561267/128573153-478225b7-80e6-4d8c-b1ca-dbbabea1fc4f.png)
![without curios](https://user-images.githubusercontent.com/88561267/128573154-c9a25d60-2481-488f-b80d-56e13640aca2.png)

**Comment by TheIllusiveC4** [comment]
Are you able to reproduce this with just Curios and JEI installed?

**Comment by Ceimnithe** [comment]
Ugh, I should have thought of that, my bad. Apparently not - loaded up a game with only JEI and Curios and the issue was fixed. Added everything back in afterwards and couldn't reproduce the issue. In the meantime, some mods got updated and I tweaked a few as I've been playtesting my setup, so I'm guessing the issue resolved itself somewhere along the line. I'll keep testing things and reopen if I can reproduce it. Otherwise, thank you for your time!
